tests/app/views/contexts/test_summary_context.py

```diff
@@ -541,7 +551,10 @@ def test_context_for_driving_question_summary_empty_list():
         {},
     )
 
-    context = summary_context(current_location)
+    context = summary_context(
+        current_location,
+        routing_path=RoutingPath(["anyone-usually-live-at"], section_id="section"),
```

Comment:
```suggestion
```

Reply:
Removing routing path will break the test as it's required positional argument.